runtimes/aleph-debian-11-python/init1.py

```diff
@@ -247,6 +247,7 @@ async def setup_code_asgi(code: bytes, encoding: Encoding, entrypoint: str) -> A
         module = __import__(module_name)
         for level in module_name.split(".")[1:]:
             module = getattr(module, level)
+        logger.debug("import done")
```

Comment:
Should this be kept ?

Reply:
The underlying problem isn't really fixed so it will helps when the issue occur again.